auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -121,9 +122,9 @@ public String bdreLogin(HttpSession session,
     }
 
     @RequestMapping(value = "/validatetoken/{authtoken}", method = RequestMethod.GET)
-    public
+
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)